README.md

```diff
@@ -298,7 +298,7 @@ run:
 
 If you have any questions we're in #docker-machine on Freenode.
 
-=======
+=========
```

Comment:
@ehazlett : you're right it seems, i wanted ot be sure that i wasn't breaking something (even tought mardwon seems not being code :) ).
I'm going to update the PR to delette this one.